scripts/mvf_rechunk.py

```diff
@@ -38,6 +38,13 @@ def __init__(self, arg):
             raise ValueError('Chunk sizes must be positive')
 
 
+def _make_lost(data):
+    if all(stride == 0 for stride in data.strides):
```

Comment:
Are you still using the cunning zero strides?

Reply:
Ah crap, I thought I'd reverted things here too. Let me see what went wrong.

Reply:
Ok, I've force-pushed the right version now (the only changes are in mvf_rechunk.py). Sorry for the inconvenience.